### DISCUSSION TITLE
how to Custom hotkey actions

### DISCUSSION THREAD

**Discussion by Fang-zhixian** [seerge/g-helper#3030] [Q&A]
"
Select Custom next to appropriate hotkey under Extra settings and do one of the following:

To run any custom application - put a full path to exe into action text field, preferably in quotes, for example:
"C:\Program Files\EA Games\Battlefield 2042\BF2042.exe"

To simulate any windows key or key-combination - put appropriate keycode(s) into the "action" field separated by space. For example 0x2C for Print Screen or 0x11 0xA0 0x31 for Ctrl+Shift+1
"
The above guideline describes how to customize hotkey actions, but I am not entirely clear on how to proceed. For example, if I want to set the M1 key to open a specific folder, how should I configure this? If this setup is correct, how can I use the M1 key to launch the folder?
![image](https://github.com/user-attachments/assets/748cf1fe-c333-48f4-90a4-a7eb8a6d2914)

**Comment by seerge** [comment]
@Fang-zhixian if you want to open windows explorer with specific folder in it, you need to launch explorer app then :) https://www.howtogeek.com/691201/how-to-open-file-explorer-using-command-prompt-on-windows-10/#:~:text=Run%20%22start%20.%22%20in%20a,Explorer%20to%20a%20specific%20folder.

**Reply by Fang-zhixian** [reply]
I am wondering how I can use hotkeys to open specific folders or perform certain actions.

**Reply by seerge** [reply]
I just answered you above

To open folder - you need to launch windows explorer app with that folder as explained in the link
